xml/repeated-content-decl.ent

```diff
@@ -0,0 +1,118 @@
+<!--...........................................................................
+Repeated content entities, for re-printing content in multiple locations
+rather than using xrefs, so that users do not have to skip around so much
+This works just like copy-paste, so be mindful of sections, titles, and
+paragraphs
+............................................................................-->
+
+<!-- Use xmlns="http://docbook.org/ns/docbook" on the first occurrence of every
+formatting tag in each entity, to avoid "Namespace default prefix was not found" 
+errors -->
+
+<!ENTITY namespaces 
+'<formalpara xmlns="http://docbook.org/ns/docbook">
+<title>Length of release names</title> 
+<para xmlns="http://docbook.org/ns/docbook">
```

Comment:
You can remove the DocBook namespace here as you already included it into `formalpara` (which is its parent element).

If you define the default namespace on its root element (in our example `formalpara`) it will be inherited down its child elements.